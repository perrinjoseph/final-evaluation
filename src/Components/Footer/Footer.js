import React from "react";
import Facebook from "../../images/facebook.svg";
import Linkedin from "../../images/linkedin.svg";
import Pinterest from "../../images/pinterest.svg";
import Email from "../../images/email.svg";

function Footer() {
  return (
    <div class="footer">
      <div className="footer__row">
        <img className="social__icon" src={Facebook}></img>
        <img className="social__icon" src={Linkedin}></img>
        <img className="social__icon" src={Pinterest}></img>
        <img className="social__icon" src={Email}></img>
      </div>
      <div class="col-sm-2 col-md-2 text-center">
        <div class="portfolio-footer">
          <h4>Portfolio</h4>
          <ul>
            <li>Project1</li>
            <li>Project2</li>
            <li>Project3</li>
            <li>See All</li>
          </ul>
        </div>
      </div>
    </div>
  );
}

export default Footer;
