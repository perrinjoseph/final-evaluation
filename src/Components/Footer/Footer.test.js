import { render } from "@testing-library/react";
import Footer from "./Footer";
describe("Footer", () => {
  it("should match the snapshot", () => {
    const { asFragment } = render(<Footer>Snapshot Footer</Footer>);
    expect(asFragment()).toMatchSnapshot();
  });
});
