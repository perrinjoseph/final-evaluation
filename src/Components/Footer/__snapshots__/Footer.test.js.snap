// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match the snapshot 1`] = `
<DocumentFragment>
  <div
    class="footer"
  >
    <div
      class="footer__row"
    >
      <img
        class="social__icon"
        src="facebook.svg"
      />
      <img
        class="social__icon"
        src="linkedin.svg"
      />
      <img
        class="social__icon"
        src="pinterest.svg"
      />
      <img
        class="social__icon"
        src="email.svg"
      />
    </div>
    <div
      class="col-sm-2 col-md-2 text-center"
    >
      <div
        class="portfolio-footer"
      >
        <h4>
          Portfolio
        </h4>
        <ul>
          <li>
            Project1
          </li>
          <li>
            Project2
          </li>
          <li>
            Project3
          </li>
          <li>
            See All
          </li>
        </ul>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
